File: labs/lab01/Exercise01.java
import java.util.Arrays;

public class Exercise01 {

  /**
   * @param numbers El arreglo de enteros.
   * @return La suma de los elementos del arreglo
   */
  public static int sumArray(int[] numbers) {
    return Arrays.stream(numbers).sum();
  }

  public static void main(String[] args) {
    int[] numbers = {10, 20, 30, 40};
    int total = sumArray(numbers);
    System.out.println("La suma es: " + total);
  }
}

File: labs/lab01/exercise_01.cpp
#include <iostream>
#include <vector>
#include <numeric>

int sumArray(const std::vector<int>& numbers) {
  return std::accumulate(numbers.begin(), numbers.end(), 0);
}

int main() {
  std::vector<int> numbers = {10, 20, 30, 40};
  int total = sumArray(numbers);
  std::cout << "La suma es: " << total << std::endl;

  return 0;
}
